code/game/turfs/turf.dm

```diff
@@ -257,6 +253,10 @@
 		else
 			lighting_clear_overlays()
 
+	W.levelupdate()
+	W.CalculateAdjacentTurfs()
```

Comment:
This returns the problem of RCDing a floor not causing turfs to update. Really need to find what's causing this to occur---may have to leave this as air_update_turf(1) if this can't be found.